src/types/index.ts

```diff
@@ -13,10 +16,14 @@ export {
 } from './Story';
 
 // Secondary entities
-export { default as Category } from './Category';
-export { default as Culture } from './Culture';
-export { default as OEmbedInfo } from './OEmbedInfo';
-export { default as Pagination } from './Pagination';
+export {
+    UploadcareFile,
+    UploadcareImage,
+    OEmbedInfo,
+    Pagination,
+    SelectionMode,
+    SelectionValue,
+} from './secondary';
```

Comment:
I never saw this pattern before.
What is a "secondary" entity?
Why does that distinction help with?

Reply:
Same reason *namespaces* or *modules* exist -- you group similar entities and separate those that differ. 

Contacts, Categories, Coverage, Newsroom and Stories -- they all are entities -- [first-class citizens](https://en.wikipedia.org/wiki/First-class_citizen) in our domain model -- they have their own API endpoints, they can be retrieved, updated and deleted by itself.

While *Pagination*, *Selection*, *OEmbedInfo* and *Uploadcare* files are not *entites*, but *value objects* --  *second-class objects* or just "secondary". They don't have they own APIs, they cannot be deleted or updated, they only appear as part of other entities APIs.

Reply:
Ok, I get it - we have **entities** and **other**. 

I think that whatever word is used for **other**, it should (as much as possible):
1. explain the purpose
2. indicate that **entities** do not belong there
3. be an antonym of "entities" (well, it should be enough to just have a different meaning)

With "secondary" I ask myself: "secondary what?". And since there are **entities**, it makes me think they are "secondary entities", which is not the case.
I guess it's because I didn't figure out the "first-class citizen" analogy (maybe because I have never heard "second-class citizen").

Anyway, my failure to understand this leads me to these suggestions which I believe are a bit better than `types/secondary`:
- `types/common` (my pick)
- `types/utils` 
- `types/helpers`

Reply:
As of me, I think `secondary` is more specific, as *common* is pretty much overused everywhere. Also the top-level `types` folder is *common* by its own already. 

But I have renamed it to `common` just to not overspend here. Having some sub-folder is better than not having it. So it's an improvement already.

Thanks!